core/src/mast/mod.rs

```diff
@@ -653,7 +650,7 @@ impl MastForest {
 // TEST HELPERS
 // ================================================================================================
 
-#[cfg(test)]
+#[cfg(feature = "testing")]
```

Comment:
Consider using `#[cfg(any(test, feature = "testing"))]` instead of just `#[cfg(feature = "testing")]`.

The current approach means that unit tests in the `core` crate itself cannot use `all_decorators()` when running `cargo test` without explicitly enabling the `testing` feature.

Using `any(test, feature = "testing")` allows:
- Unit tests in the same crate to use it automatically (via `test`)
- Other crates to use it when they enable the `testing` feature

This is the standard pattern for test helpers that need to be shared across crates while still being available for local unit tests.